api/routes/monitoring_routes.py

```
@@ -0,0 +1,135 @@
+        # Run the systemctl command to check if the service is active
+        result = subprocess.run(
+            ["systemctl", "is-active", service_name],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            text=True,
+        )
+
+        # Check the output; 'active' indicates the service is running
+        return result.stdout.strip() == "active"
+
+    except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)